scripts/qiita

```diff
@@ -289,6 +294,49 @@ def update_preprocessed_data(sl_out_dir, study,  preprocessed_data):
                                             preprocessed_data)
     click.echo("Preprocessed data %s successfully updated" % ppd.id)
 
+# #############################################################################
+# PORTAL COMMANDS
+# #############################################################################
+
+
+@portal.command(name="add-analyses")
+@click.option('--portal', required=True, type=str,
+              help='Portal to add the analyses to')
+@click.option('--analyses', required=True, type=str,
```

Comment:
ok, so with `nargs=-1` the command looks like: 

`qiita db portal add-analyses -p lolercoaster 1 2 3 4 5 6 7 8 9 ...